sycl/include/sycl/ext/intel/experimental/fpga_kernel_attributes.hpp

```diff
@@ -0,0 +1,154 @@
+//==----- fpga_kernel_attributes.hpp - SYCL properties associated with FPGA
+// kernel attributes ---==//
```

Comment:
The extension has name sycl_ext_intel_fpga_kernel_interface_properties. Better to rename the file from fpga_kernel_attributes.hpp to fpga_kernel_interface_properties.hpp

Reply:
added

Reply:
Sorry, looks like this wasn't applied

Reply:
You are right.  The comment was in the wrong place.  I will change the name in the next push, but I am just struggling with ensuring my test is working after the file rename

Reply:
**now updated**